File: LB/PROgram0042.c
/*

Q42) Write a PROgram which Accept number from user and return count of Odd digits

    input :- 2395
    output :- 3

*/

#include<stdio.h>

int CheckOdd(int iNo1)
{
    int iCnt = 0;
    int iDigit = 0;

    for(; iNo1 != 0;)
    {
        iDigit = iNo1 % 10;

        if(iDigit % 2 != 0)
        {
            iCnt++;
        }

        iNo1 = iNo1 / 10;
    }

    return iCnt;
}

int main()
{
    int iValue1 = 0;
    int iRet = 0;

    printf("Enter the Number : ");
    scanf("%d",&iValue1);

    iRet = CheckOdd(iValue1);

    printf("Count of Odd Digits is : %d",iRet);

    return 0;
}

File: PM/PROgram0042.py
'''

Q2) Write a PROgram that Accept the 2 Number from user And display the greater number

'''

def ChkGreater(iNo1, iNo2):
    if iNo1 > iNo2:
        print(iNo1," is Greater Number ")

    else:
        print(iNo2,"is Greater Number ")

def main():
    print("Enter the First Number : ")
    iValue1 = int(input())

    print("Enter the Second Number : ")
    iValue2 = int(input())

    ChkGreater(iValue1,iValue2)    

if __name__ == "__main__":
    main()        